config/litellm_config.yaml

```
@@ -53,4 +54,9 @@ litellm_settings:
   num_retries: 3
   request_timeout: 30
   callbacks: rag_pipeline.pre_request_hook.rag_handler_instance
-  forward_client_headers_to_llm_api: true
\ No newline at end of file
+  forward_client_headers_to_llm_api: true
+
+# MCP servers configuration for web search tool
+mcp_servers:
+  web_search:
+    url: "http://localhost:8765/search/sk-abc-123-def-456/sse"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)